src/core/types/array.js

```diff
@@ -14,6 +14,34 @@
  * limitations under the License.
  */
 
+/**
+ * Converts an array-like object to an array.
+ * @param {?IArrayLike<T>|string} arrayLike
+ * @return {!Array<T>}
+ * @template T
+ */
+export const toArray = Array.prototype.slice.call;
+
+/**
+ * Determines if value is actually an Array.
+ * @param {*} value
+ * @return {boolean}
+ */
+export const {isArray} = Array;
```

Comment:
This was previously
```javascript
export function isArray(value) {
  return Array.isArray(value);
}
```
Can we just provide the function itself, since it's static?